app.js

```diff
@@ -80,10 +80,13 @@ app.use('/components/jquery', express.static(path.join(__dirname, 'node_modules'
 app.use('/components/magnific-popup', express.static(path.join(__dirname, 'node_modules', 'magnific-popup', 'dist')));
 app.use('/components/scrollreveal', express.static(path.join(__dirname, 'node_modules', 'scrollreveal', 'dist')));
 
-app.use('/', require('./routes/index')(express, passport));
+if (config.util.getEnv('NODE_ENV') !== 'docker') {
+  app.use('/', require('./routes/index')(express, passport));
+  app.use('/project', require('./routes/project')(express));
```

Comment:
Project List is not accessible for anonymous user any longer so this can be omitted.

Reply:
done